package.json

```
@@ -3,7 +3,8 @@
     "scripts": {
         "dev": "tsc --watch",
         "build": "tsc --strict && tsc-alias",
-        "preview": "node dist/index.js",
+        "preview": "node dist/index.js > bot-logs.txt 2>&1",
+        "preview-no-logs": "node dist/index.js",
```

This isn't a very great way of doing it if developing on windows. (I don't have a great option in that case though).
But shouldn't it be this? You'll have to test.
```suggestion
        "preview": "node dist/index.js &> bot-logs.txt &>&1",
        "preview-no-logs": "node dist/index.js",
```
Another option is to reassign console.* in the root file, so that all console.log/console.error etc. are sent to the logging file.

---

Isn't that exactly what we want? What's the point in not getting the error in our logs?

---

Nono my previous thing also does it I believe. Also it doesn't work when I had tested it.

---

Still not cross platform for development, default to no-logs. so it should be preview, and preview-with-log-file